content/en/training/_index.html

```diff
@@ -11,29 +11,30 @@
   <div class="main-section">
     <div class="call-to-action" id="cta-certification">
       <div class="logo-certification cta-image cta-image-before" id="logo-cka">
-        <img src="/images/training/kubernetes-cka-white.svg"/>
+        <img src="kubernetes-cka-white.svg"/>
       </div>
       <div class="logo-certification cta-image cta-image-after" id="logo-ckad">
-        <img src="/images/training/kubernetes-ckad-white.svg"/>
+        <img src="kubernetes-ckad-white.svg"/>
       </div>
       <div class="cta-text">
-        <h2>Build your cloud native career</h2>
+        <h2 style="font-size:2em;">Build your cloud native career</h2>
         <p>Kubernetes is at the core of the cloud native movement. Training and certifications from the Linux Foundation and our training partners lets you invest in your career, learn Kubernetes, and make your cloud native projects successful.</p>
       </div>
     </div>
   </div>
 </section>
 
-<section>
-  <div class="main-section padded">
+
+<section class="second">
+  <div class="main-section padded-bottom-banner">
     <center>
       <h2>Take a free course on edX</h2>
     </center>
   <div class="col-container">
     <div class="col-nav">
       <center>
-        <h5>
-          <b>Introduction to Kubernetes <br> &nbsp;</b>
+        <h5 class="blue-h5-white-text">
+          <b>Introduction to Kubernetes
```

Comment:
Missing a closing `</b>`. This is a part of the rendering problem with this page I _think_. Either way can't hurt to close it!